test/e2e/e2e_test.go

```diff
@@ -48,7 +49,7 @@ func TestSingleBinaryEventForChannel(t *testing.T) {
 	t.Skip("Skipping until https://github.com/google/knative-gcp/issues/486 is fixed.")
 	cancel := logstream.Start(t)
 	defer cancel()
-	e2ehelpers.SingleEventForChannelTestHelper(t, binding.EncodingBinary, e2ehelpers.SubscriptionV1beta1, "", channelTestRunner, lib.DuplicatePubSubSecret)
+	e2ehelpers.SingleEventForChannelTestHelper(context.Background(), t, binding.EncodingBinary, e2ehelpers.SubscriptionV1beta1, "", channelTestRunner)
```

Comment:
Why do we remove lib.DuplicatePubSubSecret here?

Reply:
Can you try to keep the lib.DuplicatePubSubSecret and see whether E2E tests pass or not?

Reply:
thanks for the advice, it does look like it was a mistake on my part to just remove this instead of finding a way to work with the new required interface for these options. ptal